main/parse.c

```diff
@@ -1772,6 +1775,26 @@ extern void makeFileTag (const char *const fileName)
 	}
 }
 
+static void makeEOFTag (const char *const fileName)
```

Comment:
Why not merge `makeEOFTag` into `makeFileTag` and only call the later where the first is now called?